src/sentry/runner/decorators.py

```diff
@@ -25,3 +34,22 @@ def inner(ctx, *args, **kwargs):
             configure()
         return ctx.invoke(f, *args, **kwargs)
     return update_wrapper(inner, f)
+
+
+def log_level_option(f):
+    "Give ability to configure global logging level. Must be used before configuration."
+    import click
+    from functools import update_wrapper
+
+    @click.pass_context
+    @click.option('--loglevel', '-l', default=None,
+        help='Global logging level. Use wisely.',
+        envvar='SENTRY_LOG_LEVEL',
+        type=CaseInsensitiveChoice(LOG_LEVELS))
+    def inner(ctx, *args, **kwargs):
+        level = kwargs.pop('loglevel', None)
```

Comment:
Actually, we shouldn't need to worry about clobbering from the outside since `click` is seeding the value from the env already. So we can just `os.environ['SENTRY_LOG_LEVEL'] = loglevel` without worrying about it.